kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Time.kt

```diff
@@ -34,8 +38,7 @@ public fun Duration<IsoUnit>.toSeconds(): Long {
  */
 @Suppress("MagicNumber")  // These are all time units!
 public fun java.time.Duration.toHuman(): String? {
-    val parts = mutableListOf<String>()
```

Comment:
While I appreciate the refactoring, your version of this function doesn't give the same output.